proto/healthchecks/health_checks.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2025 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package org.lfedge.eve.healthchecks;
+option go_package  = "github.com/lf-edge/eve-api/go/healthchecks";
+option java_package = "org.lfedge.eve.healthchecks";
+
+// Request payload for the POST /api/v1/edgeDevice/healthchecks endpoint.
+//
+// ZHealthCheckMsg encapsulates periodic health check reports for critical
+// system hardware components. Each message typically contains status updates
+// for key hardware elements, such as ECC memory and storage devices. These
+// reports are generated at a low frequency (e.g., once every 24 hours) and
+// are not retransmitted in case of network failure, as their data remains
+// relatively stable.
+//
+// The message is secured within a TLS session, which is bound to the device
+// certificate to ensure integrity and authenticity.
```

Comment:
Shouldn't we be using AuthContainer here where the AuthContainer is signed using the device identity?
(We don't seem to be using that for the Info and Metrics messages but it would make sense to introduce it for new messages like this one.)

Reply:
It is a good idea to use the AuthContainer. However, I need some help implementing it. Can you give me some pointers? I saw how we do it for `CompoundEdgeDevConfig` in proto/config/compound_devconfig.proto, but I am unsure how to do it in our case.

Reply:
Sure I can provide pointers. I don't know if we have sending code in EVE-OS or only have the receiving code.
For now it makes sense to update the comments in this API PR.